services/table-analyzer.js

```diff
@@ -29,12 +31,16 @@ function TableAnalyzer(queryInterface, config) {
         return 'BOOLEAN';
       case 'CHARACTER VARYING':
       case 'TEXT':
+      case (type.match(/TEXT.*/i) || {}).input:
       case (type.match(/VARCHAR.*/i) || {}).input:
       case (type.match(/CHAR.*/i) || {}).input:
         return 'STRING';
+      case 'UNIQUEIDENTIFIER':
+        return 'UUID';
       case 'JSONB':
         return 'JSONB';
       case 'SMALLINT':
+      case 'BIT':
```

Comment:
hey @tlapuste, in MSSQL, should `BIT` types be considered as `NUMBER` or `BOOLEAN`? I think it should be `BOOLEAN`.